packages/runtime/src/index.ts

```diff
@@ -251,13 +251,14 @@ export default function init(inlineConfig: RuntimeOptions = {}) {
   function observe() {
     if (observing)
       return
-    const target = html() || defaultDocument.body
-    if (!target)
+    const target = () => runtimeOptions.observer?.target ? runtimeOptions.observer.target() : (html() || defaultDocument.body);
```

Comment:
Maybe we don't need `target()` as function?

Reply:
removed the `target()` as a function, now a regular `const`

Tried to add the type defs, hope it's ok 😬